weavetest/auth.go

```diff
@@ -0,0 +1,76 @@
+package weavetest
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/iov-one/weave"
+)
+
+// Auth is a mock implementing x.Authenticator interface.
+//
+// This structure authenticates any of referenced conditions.
+// You can use either Signer or Signers (or both) attributes to reference
+// conditions. This is for the convinience and each time all signers
+// (regardless which attribute) are considered.
+type Auth struct {
```

Comment:
Shouldn't there be a few tests in this package?
At least for the auth stuff and decorator wrapping

Reply:
Done in https://github.com/iov-one/weave/pull/386/commits/137fdf5f6f64ef80759e4ec45be5c2981212964a